data/json/terrain.json

```diff
@@ -7400,6 +7417,7 @@
     "type": "terrain",
     "id": "t_sewage_pipe",
     "name": "sewage pipe",
+    "description": "This fine specimen is a corridor through which travels the excrement of society.",
```

Comment:
Sewage pipes are currently used in a lot of random places as decorative terrain. I'd suggest changing it to just being a large gauge pipe, and maybe changing the name too.

Reply:
k